### PR TITLE
Added ListExample

### PR DESCRIPTION
**Pull request by Beider**

### CODE REVIEW

File: ReplicatedListExample/ListActor.cs [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+        GameSession.OnPlayerJoinedEvent -= OnPlayerJoinedOrLeft;
+        GameSession.OnPlayerLeftEvent -= OnPlayerJoinedOrLeft;
+    }
+
+    protected virtual void OnPlayerJoinedOrLeft(int PeerId)
+    {
+        if (MDStatics.IsNetworkActive() && PeerId == GetNetworkMaster())
+        {
+            // Reset to server if master left
+            this.ChangeNetworkMaster(MDStatics.GetServerId());
```
Comment (DoubleDeez) [inline ReplicatedListExample/ListActor.cs:93]: I feel like we should make it automatic in the framework. Maybe have it in a virtual function so users can pick a fallback other than the server

File: ReplicatedListExample/ListActor.cs [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+        }
+
+        PopupMenu.GetPopup().Clear();
+        foreach (int player in GameSession.GetAllPeerIds())
+        {
+            PopupMenu.GetPopup().AddItem(player.ToString());
+        }
+    }
+
+    public void OnStringListChangedChanged()
```
Comment (DoubleDeez) [inline ReplicatedListExample/ListActor.cs]: Typo? ChangedChanged

File: ReplicatedListExample/ListActor.cs [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+        {
+            return;
+        }
+
+        String value = ItemInput.Text;
+        int[] selection = DisplayList.GetSelectedItems();
+        if (selection.Length > 0 && value != null && value != "")
+        {
+            List<String> RangeAdd = new List<string>();
+            for (int i =0; i < 5; i++)
```
Comment (DoubleDeez) [inline ReplicatedListExample/ListActor.cs]: spacing: `int i = 0;`

**Review by DoubleDeez — CHANGES_REQUESTED** [review]

File: ReplicatedListExample/ListActor.cs (hunk 1 above)
Comment (DoubleDeez) [reply ReplicatedListExample/ListActor.cs:93]: (not needed now - just a suggestion)

Comment (Beider) [reply ReplicatedListExample/ListActor.cs:93]: Well I think in normal circumstances you usually clean up nodes owned by someone who left don't you? So if you want to clean it up or if you want to change owner depends on the situation.
Perhaps we can introduce a new concept where any networked node can implement the method MDReplicatedNodeOwnerLeft() and we will look for it when the owner of the node leaves. If we find it we call it, then they can either clean it up or change the owner themselves.

**Comment by Beider** [comment]
These issues have now been fixed.

**Review by DoubleDeez — APPROVED** [review]

**Pull request merged**